python/lsst/analysis/tools/tasks/base.py

```diff
@@ -193,6 +193,12 @@ def runPlots(self, data: KeyedData, **kwargs) -> Struct:
         if "plotInfo" not in kwargs:
             kwargs["plotInfo"] = _StandinPlotInfo()
         for name, action in self.config.plots.items():
+            for selector in action.prep.selectors:
+                try:
```

Comment:
The `AttributeError` is for cases where `selector.threshold` does not exist, right?Instead of the try/except, how about 
`if "threshold" in selector.keys():
    kwargs["plotInfo"]["SN"] = selector.threshold`